tools/report-converter/README.md

```diff
@@ -246,6 +247,29 @@ report-converter -t eslint -o ./codechecker_eslint_reports ./eslint_reports.json
 CodeChecker store ./codechecker_eslint_reports -n eslint
 ```
 
+## [TSLint](https://palantir.github.io/tslint)
+[TSLint](https://palantir.github.io/tslint) is a static analysis tool for
+`TypeScript`.
+
+The recommended way of running the TSLint tool is to generate a **json** output
+file.
+
+The following example shows you how to run TSLint and store the results found
+by TSLint to the CodeChecker database.
+
+```sh
+# Run TSLint.
+tslint --format json /path/to/my/project -o ./tslint_reports.json
```

Comment:
The input of `tslint` is not a directory but a `.ts` file. Or this set of `.ts` files can be provided as a glob. See the "Quick start" of this guide: https://palantir.github.io/tslint/. It didn't work for me with a directory. But maybe I did something wrong.

Reply:
@bruntib Thanks, only the documentation was wrong so I fixed it. We already have a test case for this that for a directory it will not work.